spread.yaml

```diff
@@ -522,4 +522,44 @@ suites:
             distro_purge_package qemu genisoimage sshpass
             snap remove ubuntu-image
 
+    tests/performance/:
+        summary: Performance tests for snapd
+        manual: true
+        environment:
+            RES_FILE: "$PROJECT_PATH/test.perf"
```

Comment:
Nitpick, can you use something longer than `RES` (result, re...what?) so that it's more obvious.